index.js

```diff
@@ -111,7 +111,7 @@ module.exports.bootstrap = async ({ getPluginContext, options, refresh, setPlugi
             }
 
             // Handling deleted assets.
-            response.deletedAssets.forEach(asset => {
+            response.deletedAssets.forEach((asset) => {
```

Comment:
Changes like these were created by running the existing "format" script in the package.json, I assume this parenthesis ruling is in your stackbit prettier config